src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-prism/.vsconfig

```diff
@@ -1,48 +1,109 @@
 {
   "version": "1.0",
   "components": [
-    "microsoft.net.runtime.mono.tooling",
-    "microsoft.net.sdk.emscripten",
-    "wasm.tools",
-    "Component.OpenJDK",
-    "Component.Xamarin",
```

Comment:
@jeromelaban Everytime I build a solution filter locally, I see these `.vsconfig`s are modified automatically. Do you know if these changes are correct? Why are they made? and if they are not correct, how to stop the build from doing them?

Reply:
That I do not know. I recently noticed that with VS 17.4, could it be related to deprecated feature? Or just simply disambiguation or duplicates based on hierarchy ?

Reply:
@jeromelaban I'm not sure. But it's very likely related to this PR https://github.com/unoplatform/uno/pull/7823